lib/neo4j/migrations/migration_file.rb

```diff
@@ -0,0 +1,24 @@
+module Neo4j
+  module Migrations
+    class MigrationFile
+      attr_reader :file_name, :symbol_name, :class_name, :version
+
+      def initialize(file_name)
+        @file_name = file_name
+        extract_data!
+      end
+
+      def create
+        require file_name
```

Comment:
Not a huge deal, but I generally prefer to refer to instance variables internally (like `@file_name`), not the accessors